src/styles/blocks/main.scss

```diff
@@ -0,0 +1,20 @@
+.main {
+  margin: 0;
+  padding: 50px 40px;
+
+  display: grid;
+  justify-content: center;
+  gap: 46px 48px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
```

Comment:
Ensure media queries are ordered consistently from smaller to larger. The current order is correct, but it's important to maintain this consistency if more breakpoints are added or if the existing ones are modified.